appender.go

```diff
@@ -60,18 +60,19 @@ var (
 // server to make progress encoding while Elasticsearch is busy servicing flushed bulk requests.
 type Appender struct {
 	// legacy metrics for Stats()
-	bulkRequests          int64
-	docsAdded             int64
-	docsActive            int64
-	docsFailed            int64
-	docsFailedClient      int64
-	docsFailedServer      int64
-	docsIndexed           int64
-	tooManyRequests       int64
-	bytesTotal            int64
-	availableBulkRequests int64
-	activeCreated         int64
-	activeDestroyed       int64
+	bulkRequests           int64
+	docsAdded              int64
+	docsActive             int64
+	docsFailed             int64
+	docsFailedClient       int64
+	docsFailedServer       int64
+	docsIndexed            int64
+	tooManyRequests        int64
+	bytesTotal             int64
+	bytesUncompressedTotal int64
```

Comment:
Due to the verbose/long name introduced, the indentation has changed and the readability of diff is not so great. I tried to shorten the name to 
- `byteUncompTotal`, but it was unnatural to read,
- `bytesRawTotal` was introducing cognitive load that raw and uncompressed bytes refer to the same thing

In the end, I chose `bytesUncompressedTotal`, but any suggestions are more than welcome.